utils/types/InstallPlanMutationVariables.ts

```diff
@@ -0,0 +1,59 @@
+export interface InstallPlanMutationVariable {
+  dryRun: boolean;
+  description: string;
+  displayName: string;
+  fallback?: InstallPlanDirectiveInput;
+  heading: string;
+  id: string;
+  primary: InstallPlanDirectiveInput;
+  target: InstallPlanTargetInput;
+}
+
+export interface InstallPlanTargetInput {
+  destination: InstallPlanDestination;
+  os?: InstallPlanOperatingSystem[];
+  type: InstallPlanTargetType;
+}
+
+export type InstallPlanDirectiveInput =
+  | TargetedInstallDirective
+  | LinkDirective
+  | NerdletDirective
+  | DefaultDirective;
+
+interface TargetedInstallDirective {
+  targeted: {
+    recipeName: string;
+  };
+}
+
+interface LinkDirective {
+  link: {
+    url: string;
+  };
+}
+
+interface NerdletDirective {
+  nerdlet: {
+    nerdletId: string;
+    nerdletState: string;
+  };
+}
+
+interface DefaultDirective {
+  mode: string;
+  destination: undefined;
```

Comment:
```suggestion
  destination?: string;
```

Reply:
We need to determine whether we can actually hit the case where this type would be used. I will chat with Devex